packages/mobile/src/components/top-tab-bar/TopTabBar.tsx

```diff
@@ -102,11 +114,9 @@ export const TopTabBar = ({ state, descriptors, navigation, position }) => {
     })
   }
 
-  const tabWidth = 100 / state.routes.length
-
   const left = position.interpolate({
     inputRange: [0, state.routes.length],
-    outputRange: [0, screenWidth]
+    outputRange: [horizontalPadding, screenWidth - horizontalPadding]
```

Comment:
this is really cool!

Reply:
Yeah unfortunately we have to use that value for the animation values too. Definitely like the approach of passing things into useStyles though, will use in the future